impl_core/src/test/java/io/opencensus/implcore/stats/RecordUtilsTest.java

```diff
@@ -83,6 +92,61 @@ public void testGetTagValues() {
         .inOrder();
   }
 
+  @Test
+  public void testGetTagValues_MapDeprecatedRpcTag() {
+    List<TagKey> columns = Arrays.asList(RecordUtils.RPC_STATUS, RecordUtils.RPC_METHOD);
+    Map<TagKey, TagValueWithMetadata> tags =
+        ImmutableMap.of(
+            RecordUtils.GRPC_CLIENT_METHOD, METHOD_V_WITH_MD,
+            RecordUtils.GRPC_CLIENT_STATUS, STATUS_V_WITH_MD);
+
+    assertThat(RecordUtils.getTagValues(tags, columns))
+        .containsExactly(STATUS_V, METHOD_V)
+        .inOrder();
+  }
+
+  @Test
+  public void testGetTagValues_MapDeprecatedRpcTag_WithServerTag() {
+    List<TagKey> columns = Arrays.asList(RecordUtils.RPC_STATUS, RecordUtils.RPC_METHOD);
+    Map<TagKey, TagValueWithMetadata> tags =
+        ImmutableMap.of(
+            RecordUtils.GRPC_SERVER_METHOD, METHOD_V_WITH_MD,
+            RecordUtils.GRPC_SERVER_STATUS, STATUS_V_WITH_MD);
+
+    assertThat(RecordUtils.getTagValues(tags, columns))
+        .containsExactly(STATUS_V, METHOD_V)
+        .inOrder();
+  }
+
+  @Test
+  public void testGetTagValues_MapDeprecatedRpcTag_PreferClientTag() {
+    List<TagKey> columns = Arrays.asList(RecordUtils.RPC_STATUS, RecordUtils.RPC_METHOD);
+    Map<TagKey, TagValueWithMetadata> tags =
+        ImmutableMap.of(
+            RecordUtils.GRPC_SERVER_METHOD, METHOD_V_WITH_MD,
+            RecordUtils.GRPC_SERVER_STATUS, STATUS_V_WITH_MD,
+            RecordUtils.GRPC_CLIENT_METHOD, METHOD_V_2_WITH_MD,
+            RecordUtils.GRPC_CLIENT_STATUS, STATUS_V_2_WITH_MD);
```

Comment:
Yes - these tests clarify what's going on, thanks@ - you may want to add a test that contains GRPC_SERVER_METHOD, GRPC_CLIENT_METHOD, and RPC_METHOD to verify that the RPC_METHOD tag value does not unexpectedly get overridden by the other tags but I'll leave that up to you.

Reply:
Good idea, added one in https://github.com/census-instrumentation/opencensus-java/pull/1854/commits/9e371bafa42be0b43b7f0a9f111de0c3c9ea89db.